cmd/nerdctl/volume/volume_prune_linux_test.go

```diff
@@ -56,54 +56,58 @@ func TestVolumePrune(t *testing.T) {
 	}
 
 	// This set must be marked as private, since we cannot prune without interacting with other tests.
-	testGroup := &test.Group{
```

Comment:
This pattern does not work here, as the `Group` will run in `Parallel` with other tests, breaking things for Docker.
It works with nerdctl because we are making the subtests pivate - for Docker, the subtests disable parallel, but since the parent stays parallel, it blows up.

Changing to a `Test` parent instead of a group, and disabling parallel on the subtests.